doc/attr.xml

```diff
@@ -1774,3 +1774,85 @@ gap> ChromaticNumber(D);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DigraphCartesianProductProjections">
+<ManSection>
+  <Attr Name="DigraphCartesianProductProjections" Arg="digraph"/>
+  <Returns>A list of transformations.</Returns>
+  <Description>
+    If <A>digraph</A> is a Cartesian product digraph, 
+    <C>digraph = DigraphCartesianProduct(gr_1, gr_2, ... )</C>, then
+    <C>DigraphCartesianProductProjections</C> returns a list <C>proj</C>
+    such that <C>proj[i]</C> is the projection onto the <C>i</C>-th
+    coordinate of the product.
+
+    A projection is an idempotent endomorphism of <A>digraph</A>. If
+    <C>gr1, gr2, ...</C> are all loopless digraphs, then the restriction
+    of <A>digraph</A> onto the image of <C>proj[i]</C> is isomorphic to
+    <C>gr_i</C>.
+
+    Currently this attribute is only set upon creating a graph via
```

Comment:
This means that `DigraphCartesianProductProjections` can only ever do anything for immutable digraphs. In particular, the projections are immediately thrown away upon creation for mutable digraphs. I think this should be mentioned at least. Do you have any alternative suggestions @james-d-mitchell?

Reply:
I think it's just best to draw attention that this only works for immutable digraphs, as I will suggest in a concrete suggestion.